### DISCUSSION TITLE
Issue template editor: "Edit labels" results in lost WIP templates

### DISCUSSION THREAD

**Discussion by rzmk** [community/community#62103] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

I was adding issue templates for my GitHub repository from the GUI and was adding templates. I wanted to edit a label so under the Labels section for a new issue template there is an "Edit labels" button. I clicked it, it redirected me to the labels page. But it was on the same tab, and when I went back a page to my templates the new ones I was working on disappeared.

Here's an example:

https://github.com/community/community/assets/30333942/ba76a84a-0772-4f9d-a248-3fbd34ee60ab

Maybe opening the labels page in a new tab or some other solution would be helpful.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by rprobst13** [comment]
Yeah this bug still exists. I was creating a new issue, realized it needed a new label and then hit "edit labels" to add it. This opened in the same tab and then I lost the ticket I was writing out.

**Comment by georgeh0** [comment]
Hit by the same issue! Lost my draft issue (with a lot of content). Any way to recover?
